server/src/internalClusterTest/java/org/opensearch/snapshots/ConcurrentSnapshotsIT.java

```diff
@@ -314,11 +314,11 @@ public void testSnapshotRunsAfterInProgressDelete() throws Exception {
 
         blockMasterFromFinalizingSnapshotOnIndexFile(repoName);
```

Comment:
Method name `blockMasterFromFinalizingSnapshotOnIndexFile` -> `blockClusterManagerFromFinalizingSnapshotOnIndexFile`

Reply:
It's a public method, and it has been published to maven, so I plan to change the name in next major version (3.0), and it's tracked in the issue https://github.com/opensearch-project/OpenSearch/issues/1684